### ISSUE TITLE
Tuning hyperparameters for finetuning with custom dataset

### ISSUE THREAD

**Issue by dersugiritlioglu** [xinntao/Real-ESRGAN#315]
Hi all! I would like to finetune the network for images that would possibly include little details.

The pretrained network performs better than the finetuned network in different ways;
- If the iteration count is high, then we observe loss in details since the network thinks they are artifacts when they aren't.
- If this is not the case, resolutions seems lower in some locations in the image.

How should i tune the hyperparameters to obtain a network that gives better results with such a custom dataset than the pretrained one? Thanks in advance.
